bitsail-connectors/connector-kudu/src/main/java/com/bytedance/bitsail/connector/kudu/option/KuduReaderOptions.java

```diff
@@ -113,22 +113,18 @@ public interface KuduReaderOptions extends ReaderOptions.BaseReaderOptions {
 
   ConfigOption<Boolean> CACHE_BLOCKS =
       key(READER_PREFIX + "enable_cache_blocks")
```

Comment:
<img width="739" alt="image" src="https://github.com/bytedance/bitsail/assets/6771378/99d46ae8-c2bb-417a-b185-6accc2f47ab9">

In our use case, we do not use cache the data, because we just scan data once mostly